### ISSUE TITLE
Props validation failing

### ISSUE THREAD

**Issue by Sazarov** [codejamninja/react-gantt#16]
```
Warning: Failed prop type: Invalid prop `titleStyle` of type `object` supplied to `GanttPopup`, expected `string`.
    in GanttPopup (created by GanttRow)
    in GanttRow (created by Demo)
    in tbody (created by ReactGantt)
    in table (created by ReactGantt)
    in div (created by ReactGantt)
    in ReactGantt (created by Demo)
    in div (created by Demo)
    in Demo (created by Route)
    in Route (created by Parent)
    in Switch (created by Parent)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by Parent)
    in Parent
```

This error is thrown when hovering over the bars.

**Comment by Ajtak** [comment]
Some here

**Comment by Ajtak** [comment]
Fixed in #19
